server/main.py

```diff
@@ -133,17 +132,27 @@ async def index(request):
     if memory:
         job.memory(memory)
 
-    # NOTE: if you add an environment variable here, make sure to update
-    # the cpg_utils.hail_batch.copy_common_env function!
-    job.env('CPG_ACCESS_LEVEL', access_level)
-    job.env('CPG_DATASET', dataset)
-    job.env('CPG_DATASET_GCP_PROJECT', dataset_gcp_project)
-    job.env('CPG_DRIVER_IMAGE', DRIVER_IMAGE)
-    job.env('CPG_IMAGE_REGISTRY_PREFIX', IMAGE_REGISTRY_PREFIX)
-    job.env('CPG_REFERENCE_PREFIX', REFERENCE_PREFIX)
-    job.env('CPG_OUTPUT_PREFIX', output_prefix)
-    job.env('HAIL_BILLING_PROJECT', dataset)
-    job.env('HAIL_BUCKET', hail_bucket)
+    # Prepare the job's configuration, which will be written to a blob.
+    config = {
+        'hail': {
+            'billing_project': dataset,
+            'bucket': hail_bucket,
+        },
+        'workflow': {
+            'access_level': access_level,
+            'dataset': dataset,
+            'dataset_gcp_project': dataset_gcp_project,
+            'driver_image': DRIVER_IMAGE,
+            'image_registry_prefix': IMAGE_REGISTRY_PREFIX,
+            'reference_prefix': REFERENCE_PREFIX,
+            'output_prefix': output_prefix,
```

Comment:
`CPG_WEB_URL_TEMPLATE` added here https://github.com/populationgenomics/analysis-runner/pull/411/files is missing, might wanna merge from main :)